packages/cli/README.md

```diff
@@ -5,6 +5,14 @@ Command line interface tool for reapit
 
 ## Install
 
+Make sure you have an `.npmrc` file in your root and add a valid npm token to the registry url 
```

Comment:
Sounds a bit vague, not sure exactly how to make it clearer but maybe worth a thought